tests/integration/workflows/test_workflows_tasks_upload.py

```diff
@@ -245,6 +247,8 @@ def test_store_record_inspirehep_api_literature_new(workflow_app):
     assert workflow.data['control_number'] == expected_control_number
     assert workflow.extra_data['recid'] == expected_control_number
     assert workflow.extra_data['head_uuid'] == expected_head_uuid
+    base_url = ensure_scheme(current_app.config["SERVER_NAME"])
+    assert workflow.extra_data['url'] == join(base_url, 'record', workflow.extra_data['recid'])
```

Comment:
it's not very useful to have a test that basically contains all the same logic as the code. If for some reason it doesn't work correctly, this test will never tell you. You can read more about this in the great Google testing blog: https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html.
```suggestion
    assert workflow.extra_data['url'] == "http://localhost:5000/record/111"
```